.github/workflows/build.yml

```diff
@@ -60,16 +60,16 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: "recursive"
+      - name: "use sdk manager to install cmake"
+        run: /usr/local/android-sdk/cmdline-tools/tools/bin/sdkmanager  --install "cmake;3.22.1"
+      - name: "Update the path"
+        run: echo "/usr/local/android-sdk/cmake/3.22.1/bin" >> $GITHUB_PATH
       - name: "Run generate script"
         run: ./bldsys/scripts/generate_android_gradle.sh
       - name: "Explicitly setup SDK path with local.properties."
         run: |
           echo "sdk.dir=/usr/local/android-sdk" > build/android_gradle/local.properties
           echo "ndk.dir=/usr/local/android-ndk" >> build/android_gradle/local.properties
```

Comment:
maybe we do not have to do this to local.properties anymore: let us change the build.gradle to specify which version to use.